README.md

```diff
@@ -22,24 +22,9 @@ Hack Club was founded by [a teenager](https://zachlatta.com) who dropped out of
 | Use our logos & banners         | <https://hackclub.com/brand/>        |
 | Follow us on Twitter            | <https://twitter.com/hackclub>       |
 
-## Contributing to our [Workshops](https://workshops.hackclub.com/)
+## Contributing
 
-Contributions are welcome!
-
-If you need any help, please contact us at team@hackclub.com or on our [Slack](https://slack.hackclub.com/).
-
-1. Check out our [public issues board][0]. If your issue isn't on the board, [open a new one][1].
```

Comment:
Can we keep some of these directions that are still relevant? (Not this first item.) Personally I'm a lot less likely to click through & read another file